_resources/storage.md

```diff
@@ -1,12 +1,38 @@
 ---
-title: NextCloud and S3
-description: NextCloud
+title: Distributed Storage
+description: Storage
 name: storage
 layout: default
 ---
 
 {% include layouts/title.md %}
 
+#### S3
+The main storage for EIC-related files is on S3 interface, for EIC use.
```

Comment:
There are two storage solutions for EIC use, of equal size and importance: S3 and XRootD. We should modify this and remove "main".